addons/mail_plugin/tests/test_controller.py

```diff
@@ -187,3 +187,10 @@ def test_get_partner_no_email_returned_by_iap(self):
         )
         second_company_id = result["partner"]["company"]["id"]
         self.assertEqual(first_company_id, second_company_id, "Should not create a new company")
+
+    def test_get_partner_notification_email(self):
+        """The EMAIL FROM is the notification email."""
```

Comment:
Sugg: """ When the email_from is the server default from address, we return a custom message instead of trying to match a partner record """